.github/workflows/unittest.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         if: ${{ matrix.os == 'ubuntu-latest' || matrix.os == 'macOS-latest' }}
 
       - name: "Upgrade pip"
-        run: pip install --upgrade pip
+        run: python -m pip install --upgrade pip
 
       - name: "Install package"
         run: pip install -e .
```

Comment:
how would you feel about changing this one at the same time? then we don't rely on a `pip` binary existing anywhere sensible, and we don't have weird undocumented differences in the way pip's being invoked within the same file.

Reply:
Sounds like a good idea, I'll add that now